parser.ts

```diff
@@ -695,14 +699,25 @@ export function traverseStmt(c: TreeCursor, s: string): Stmt<Location> {
       return { tag: "break", a: location };
     case "ForStatement":
       c.firstChild(); // Focus on for
+      var targets: AssignTarget<Location>[] = [];
       c.nextSibling(); // Focus on variable name
       let name = s.substring(c.from, c.to);
+      let ass: Assignable<Location> = { tag: "id", name: name };
+      targets.push({
+        target: ass,
+        starred: false,
+        ignore: false,
+      });
       c.nextSibling(); // Focus on in / ','
-      var index = null;
       if (s.substring(c.from, c.to) == ",") {
-        index = name;
         c.nextSibling(); // Focus on var name
         name = s.substring(c.from, c.to);
+        ass = { tag: "id", name: name };
+        targets.push({
+          target: ass,
+          starred: false,
+          ignore: false,
+        });
         c.nextSibling(); // Focus on in
       }
```

Comment:
Hi @Quincy0v0 , in this section, does this mean that the for loop doesn't support arbitrary de-structuring (across `n` targets), and instead only supports de-structuring up to `2` targets (which I'm assuming is cases like `index, name` for `enumerate`)?

Reply:
Ahh this is a thing. I would just change that into a `while` loop so I could parse all targets. That would be an easy fix though

Reply:
This would be the right approach, right @R167 ?

Reply:
That sounds great -- definitely makes sense. Do you know which team's PR would implement this?

Reply:
I would say you do it since you need it? Honestly, I might just make it a required value rather than optional. Currently only being used in `traverseAssignment`. The only other people I could think might need it would be comprehensions.

Reply:
I've changed the parser and it should work now!

Reply:
> I've changed the parser and it should work now!

Some reason i'm not seeing these changes to `traverseDestructure` in the diff(?)